modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -51,7 +52,6 @@ class ImagingUploader extends Component {
         // hiddenHeaders state and pass this.state.hiddenHeaders as a prop to
```

Comment:
this FIXME comment starting on line 51 can be removed because you just made the fix